app/views/projects/index.html.erb

```diff
@@ -1,4 +1,9 @@
-<h1 class="display-4 mt-4 ml-5 d-inline-block">Section: <%= @section.number %> - <%= @section.semester %> <%= @section.year %></h1>
+<h1 class="display-4 mt-4 ml-5 d-inline-block">Section: <%= @section.number %> - <%= @section.semester %> <%= @section.year %></h1><br>
+<h2 class="display-5 mt-4 ml-5 d-inline-block">Instructor: 
```

Comment:
The buttons are out of line now, look at master if you need to see how'd they looked. They used to be in line with the "Section: 501 ..." Maybe move this line below the Roster, edit, delete buttons?